addons/account/models/account_move.py

```diff
@@ -3554,7 +3554,7 @@ def _onchange_account_id(self):
         ''' Recompute 'tax_ids' based on 'account_id'.
         /!\ Don't remove existing taxes if there is no explicit taxes set on the account.
         '''
-        if not self.display_type and (self.account_id.tax_ids or not self.tax_ids):
+        if not self.display_type and self.account_id.tax_ids:
```

Comment:
I don't understand this change; can you not force the taxes after having set the account?